docs/api/clipboard.md

```diff
@@ -1,7 +1,6 @@
 # clipboard
 
-The `clipboard` provides methods to perform copy and paste operations. The following example
-shows how to write a string to the clipboard:
+The `clipboard` object provides methods to perform copy and paste operations. The following example shows how to write a string to the clipboard:
```

Comment:
I think module is slightly better since you always access it via a `require` statement.